### PR TITLE
32864 Migration Script Used on Forms Component

### PR DESCRIPTION
**Pull request by pdavies88**
## Description
Segmented Progress Bar Migration script should be used on forms

## Original issue(s)
Part of ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/32864 

## Testing done
Screen Shot 2021-12-15 at 12 29 53 PM

## Screenshots
Screen Shot 2021-12-15 at 12 28 53 PM
Screen Shot 2021-12-15 at 2 19 47 PM
Screen Shot 2021-12-15 at 2 22 48 PM
Screen Shot 2021-12-16 at 9 37 23 AM

## Acceptance criteria
 - [x] Run migration script in src/platform/forms
 - [x] Run migration script in src/applications/letters
 - [x] Open PR to merge forms changes and merge letters changes

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/forms-system/src/js/components/FormNav.jsx
Comment (bkjohnson) [inline src/platform/forms-system/src/js/components/FormNav.jsx:98]: Can we get a browser screenshot of what a form page looks like with this in the DOM?

Comment (pdavies88) [reply src/platform/forms-system/src/js/components/FormNav.jsx:98]: I updated it in the Screenshots section of the main thread

**Review by bkjohnson — COMMENTED** [review]
This looks fine, but I'm going to hold off on approving until we get the fix for the `ariaLabel` merged into the component definition. Once that's been published we can probably just update the `component-library` version in here.

**Review by Mottie — APPROVED** [review]
Thank you!

File: src/platform/forms-system/src/js/components/FormNav.jsx
Comment (Mottie) [reply src/platform/forms-system/src/js/components/FormNav.jsx:98]: This document page will need to be updated as well - https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/forms/available-features-and-usage-guidelines/#progress-bar

Comment (pdavies88) [reply src/platform/forms-system/src/js/components/FormNav.jsx:98]: Thanks @Mottie I have created the corresponding PR here: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/713

**Review by andresriveratoro — APPROVED** [review]

**Review by andresriveratoro — APPROVED** [review]

**Comment by andresriveratoro** [comment]
Working as expected with the latest commit. 
Screen Shot 2021-12-16 at 13 11 40

**Review by alexandec — APPROVED** [review]

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**
